scout/server/links.py

```diff
@@ -607,3 +608,18 @@ def spidex_human(variant_obj):
         return "medium"
 
     return "high"
+
+
+def external_primer_order_link(variant_obj, build=None):
```

Comment:
Oh, and this wants to have a docstring!

Reply:
I added docstrings for this function, and for some other functions in the same file!